dnsdist: compiler warning in sodcrypto.cc

dnsdist ae1fd3377

### Short description

```
sodcrypto.cc:60:67: warning: adding 'size_type' (aka 'unsigned long') to a string does not append to the string [-Wstring-plus-int]
    throw std::runtime_error("Could not decrypt message of size " + msg.length());
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~
sodcrypto.cc:60:67: note: use array indexing to silence this warning
    throw std::runtime_error("Could not decrypt message of size " + msg.length());
                                                                  ^
                             &                                    [             ]
```



### Environment
% clang --version
Apple LLVM version 9.0.0 (clang-900.0.39.2)
Target: x86_64-apple-darwin17.4.0

--- a/pdns/sodcrypto.cc
+++ b/pdns/sodcrypto.cc
@@ -57,7 +57,7 @@ std::string sodDecryptSym(const std::string& msg, const std::string& key, Sodium
   std::string decrypted;
 
   if (msg.length() < crypto_secretbox_MACBYTES) {
-    throw std::runtime_error("Could not decrypt message of size " + msg.length());
+    throw std::runtime_error("Could not decrypt message of size " + std::to_string(msg.length()));
   }
 
   decrypted.resize(msg.length() - crypto_secretbox_MACBYTES);

